UnitTests/UnitTests.Shared/Helpers/Test_EmbeddedResources.cs

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.Toolkit.Helpers;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace UnitTests.Helpers
+{
+    [TestClass]
+    public class Test_EmbeddedResources
```

Comment:
Should we have a test of loading an image file with GetStream and checking for a pixel value or something?

Reply:
Sure thing! Though I'd recommend not reviewing this yet as I'm not really sure about how to properly structure this (that's why I reverted this to draft). .NET Native makes it impossible to have it setup like I wanted, and with an extension method for `Assembly` it would be less intuitive to use and we'd also potentially have some naming conflicts with existing PRs. Will need to think about how to properly expose this new API given these constraints 🤔